benchmarking/platforms/platform_base.py

```diff
@@ -78,9 +78,12 @@ def preprocess(self, *args, **kwargs):
     def copyFilesToPlatform(self, files, target_dir=None):
         target_dir = (self.tgt_dir if target_dir is None else target_dir)
         if isinstance(files, string_types):
-            target_file = os.path.join(target_dir, os.path.basename(files))
-            self.util.push(files, target_file)
-            return target_file
+            if not os.path.isdir(files):
```

Comment:
why files can be a directory?

Reply:
Sometimes, we need to create a tmp directory in the run time for storing the application files.

Reply:
Let's discuss this use case.